pkg/storage/stores/indexshipper/compactor/deletion/request_handler.go

```diff
@@ -21,19 +20,17 @@ const deletionNotAvailableMsg = "deletion is not available for this tenant"
 
 // DeleteRequestHandler provides handlers for delete requests
 type DeleteRequestHandler struct {
-	deleteRequestsStore       DeleteRequestsStore
-	metrics                   *deleteRequestHandlerMetrics
-	limits                    retention.Limits
-	deleteRequestCancelPeriod time.Duration
+	deleteRequestsStore DeleteRequestsStore
```

Comment:
Remove the cancellation period from the API. This allows us to delete any pending request at any time.

Reply:
I think this comment is attached to the wrong place?